smart-contracts/test/Lock/erc721/enumerable.js

```diff
@@ -0,0 +1,47 @@
+const deployLocks = require('../../helpers/deployLocks')
+const shouldFail = require('../../helpers/shouldFail')
+
+const unlockContract = artifacts.require('../Unlock.sol')
+const getProxy = require('../../helpers/proxy')
+
+let unlock, locks, lock
+
+contract('Lock / erc721 / approve', accounts => {
+  before(async () => {
+    unlock = await getProxy(unlockContract)
+    locks = await deployLocks(unlock, accounts[0])
+    lock = locks['FIRST']
+
+    // Buy test keys for each account
+    const keyPrice = await lock.keyPrice()
+    for (let i = 0; i < 5; i++) {
+      await lock.purchase(accounts[i], web3.utils.padLeft(0, 40), {
+        value: keyPrice.toString(),
+        from: accounts[i],
+      })
+    }
+  })
+
+  it('tokenByIndex is a no-op', async () => {
```

Comment:
Can you tell me briefly what this "no-op" is all about?

Reply:
Ah I just mean that it's expected to return the parameter it was given here, unmodified.